Assets/SuperCharacterController/SuperCharacterController/Core/SuperCharacterController.cs

```diff
@@ -648,8 +648,13 @@ public void ProbeGround(Vector3 origin, int iter)
                 // it is connected to at it's base, if there exists any
                 if (Vector3.Angle(nearHit.normal, up) > superColType.StandAngle || nearHit.distance > Tolerance)
                 {
-                    var col = nearHit.collider.gameObject.GetComponent<SuperCollisionType>();
-
+                    SuperCollisionType col = null;
+                
+                    if (nearhit.collider != null)
```

Comment:
This typo should be fixed by commit  1023aac (fixed typo in nearHit). Is it not? If I go to my Code or to "Files Changed" I see the fixed Version.